frontend/public/components/persistent-volume.jsx

```diff
@@ -48,12 +50,48 @@ const Row = ({obj}) => <div className="row co-resource-list__item">
   </div>
 </div>;
 
-const Details = ({obj}) => <React.Fragment>
-  <div className="co-m-pane__body">
-    <SectionHeading text="PersistentVolume Overview" />
-    <ResourceSummary resource={obj} podSelector="spec.podSelector" showNodeSelector={false} showPodSelector />
-  </div>
-</React.Fragment>;
+const Details_ = ({obj: pv}) =>{
+  const storageClassName = _.get(pv, 'spec.storageClassName');
+  const pvcName = _.get(pv, 'spec.claimRef.name');
+  const namespace = _.get(pv, 'spec.claimRef.namespace');
+  const storage = _.get(pv, 'spec.capacity.storage');
+  const accessModes = _.get(pv, 'spec.accessModes');
+  const volumeMode = _.get(pv, 'spec.volumeMode');
+  const reclaimPolicy = _.get(pv, 'spec.persistentVolumeReclaimPolicy');
+  return (
+    <div className="co-m-pane__body">
+      <SectionHeading text="PersistentVolume Overview" />
+      <div className="row">
+        <div className="col-sm-6">
+          <ResourceSummary resource={pv} />
+          <dt>Reclaim Policy</dt>
```

Comment:
This should go inside the `ResourceSummary` component

Reply:
`Resource Summary` is a component used for variety of resource types such as configMaps, pvs etc. If I change the component, wouldn't that affect all uses of it?

Reply:
No, you don't have to change ResourceSummary. Just make it a child

```jsx
<ResourceSummary resource={pv}>
  <dt>Reclaim Policy</dt>
  <dd>{reclaimPolicy}</dd>
</ResourceSummary>
```

Reply:
Sorry if I wasn't clear